test/evaluate/CMakeLists.txt

```diff
@@ -135,6 +135,15 @@ add_test(RESHAPE reshape-test)
 add_test(ISO-binding ISO-Fortran-binding-test)
 add_test(folding folding-test)
 
+set(TEST_LIBPGMATH "-pgmath=false")
+if (LIBPGMATH_DIR)
+  find_library(LIBPGMATH pgmath PATHS ${LIBPGMATH_DIR})
+  if(LIBPGMATH)
+  set(TEST_LIBPGMATH "-pgmath=true")
```

Comment:
This seems like a much better way to manage the tests.

This line should be indented two more spaces.

Reply:
Fixed, thanks. I wish we had a clang-format equivalent for `CMakeLists.txt` and  `.sh`